test/prism_regression/assign_to_constant.rb

```diff
@@ -73,9 +78,13 @@ def method1
 end
 
 def method2
-  ConstantPath::DynamicConstant2 = "This should raise a SyntaxError at runtime"
+  # * `ruby -c --parser=parse.y` reports dynamic constant assignments for all of them.
+  # * `ruby -c --parser=prism`   only reports the first one.
+  # * At runtime, Ruby will issue warnings for all of these.
```

Comment:
Super nit: in `method1` we use future tense but here we use present (?) tense. Can we make them consistent?